cmd/registry-disk-v1alpha/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#
+# This file is part of the kubevirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2017 Red Hat, Inc.
+#
+
+FROM debian:sid
+
+MAINTAINER "David Vossel" \<dvossel@redhat.com\>
+ENV container docker
+
+RUN apt-get update && apt-get install -y \
+	tgt bash curl bzip2 qemu-utils && \
+	mkdir -p /disk
```

Comment:
Adding `rm -rf /var/lib/apt/lists/*` should also save us a little bit of disk space